### ISSUE TITLE
Formatting map list

### ISSUE THREAD

**Issue by Eldenroot** [Sama34/Additional-Settings-For-Custom-Profile-Fields#4]
![image](https://user-images.githubusercontent.com/778890/79066487-824afa80-7cb8-11ea-8dc2-a13488c544b4.png)

I think it should be a popup window where you can set your output for every option (I expect it is same as in XThreads)... right?

**Comment by Eldenroot** [comment]
There should be a popup window for custom setting
![image](https://user-images.githubusercontent.com/778890/79066530-d35aee80-7cb8-11ea-902d-52fdd01d0c4d.png)

**Comment by Sama34** [comment]
Yes, however I'm not a fancy dev and MyBB doesn't offer helpers for it. I could add it eventually but I'm more concerned with the plugin working.

**Comment by Eldenroot** [comment]
still valid... also sometimes all content in these fields is lost on save 👎 

@Sama34

**Comment by Sama34** [comment]
> also sometimes all content in these fields is lost on save 👎

How so? I can't see any reason for it in the current code.
